async & performance/apB.md

```diff
@@ -96,15 +97,15 @@ ASQ( "http://some.url.1" )
 } );
 ```
 
-The iterable sequence expresses a sequential series of (sync or async) steps that looks awfully similar to a Promise chain -- in other words, it's much cleaner looking than just plain nested callbacks, but not quite as nice as the `yield`-based sequential syntax of generators.
+A sequência iterável expressa uma série sequêncial de passos (síncronos ou assíncronos) que aparentam ser extremamente similares a um encadeamento de Promises, em outras palavras, são muito mais limpos que apenas callbacks puramente aninhados, mas não tão bons como a sintaxe sequêncial de `yield`s de generators.
 
-But we pass the iterable sequence into `ASQ#runner(..)`, which runs it to completion the same as if it was a generator. The fact that an iterable sequence behaves essentially the same as a generator is notable for a couple of reasons.
+Nós passamos a sequência iterável no `ASQ#runner(..)`, que roda até sua complitude igual aos generators. O fato de que uma sequência iterável se comporta essêncialmente da mesma forma que generators e chamam atenção por uma série de razões.
```

Comment:
```suggestion
Nós passamos a sequência iterável no `ASQ#runner(..)`, que roda até sua complitude, da mesma forma como seria com generators. O fato de que uma sequência iterável se comporta essencialmente da mesma forma que generators chama a atenção por uma série de motivos.
```

pensei em substituir "chama a atenção" por "é notória" mas acho que "chama a atenção" é mais popular e bacana então ++